extensions/wikia/Chat2/ChatEntryPoint.class.php

```diff
@@ -111,17 +148,16 @@ static public function getChatUsersInfo() {
 							// member since
 							$chatter[ 'showSince' ] = $chatter[ 'editCount' ] != 0;
 							if ( $chatter[ 'showSince' ] ) {
+								$months = $wgLang->getMonthAbbreviationsArray();
 								$date = getdate( strtotime( $stats[ 'date' ] ) );
+
 								$chatter[ 'since_year' ] = $date[ 'year' ];
 								$chatter[ 'since_month' ] =  $date[ 'mon' ];
+								$chatter[ 'since' ] = sprintf( '%s %s', $months[ $chatter[ 'since_month' ] ] , $chatter[ 'since_year' ] );
 							}
 
-							if ( !empty( $wgEnableWallExt ) ) {
-								$chatter[ 'profileUrl' ] = Title::makeTitle( NS_USER_WALL, $chatter[ 'username' ] )->getFullURL();
-							} else {
-								$chatter[ 'profileUrl' ] = Title::makeTitle( NS_USER_TALK, $chatter[ 'username' ] )->getFullURL();
-							}
-
+							$profileUrlNs = !empty( $wgEnableWallExt ) ? NS_USER_WALL : NS_USER_TALK;
+							$chatter[ 'profileUrl' ] = Title::makeTitle( $profileUrlNs, $chatter[ 'username' ] )->getFullURL();
 							$chatter[ 'contribsUrl' ] = SpecialPage::getTitleFor( 'Contributions', $chatter[ 'username' ] )->getFullURL();
```

Comment:
this part of code was also refactored by @wladekb and I didn't wanna mess up here not to duplicate work.